test/handlebars/express.test.js

```diff
@@ -0,0 +1,45 @@
+/*global it, describe, before*/
+'use strict';
+
+const request = require('supertest');
+const app = require('../fixtures/app/main');
+
+describe('express handlebars setup', function () {
+	before(function () {
+		return app.promise;
+	});
+
+	it('should do templating', function (done) {
+		request(app)
+			.get('/templated')
+			.expect(200, /FT/, done);
+	});
+
+	it('should not inherit any markup by default', function (done) {
+		request(app)
+			.get('/templated')
+			.expect(200, /^<h1>FT - on/, done);
+	});
+
+	it('should support loading partials via bower', function (done) {
+		request(app)
+			.get('/templated')
+			.expect(200, /End of dep 2 partial/, done);
+	});
+
+	it('should treat undefined flags as offy (like falsey)', function (done) {
+		request(app)
+			.get('/templated')
+			.expect(200, /<undefinedflag-on><\/undefinedflag-on>/, done);
+	});
+
+	describe('content helpers', function () {
+		it('should provide a string concatenation helper', function (done) {
+			request(app)
+				.get('/templated')
+				.expect(200, /Concat onetwothree/, done);
```

Comment:
I wonder if there is a way of writing this test in a way where you can clearly see the input (i.e. `one two three`) without having to cross-reference another file (`test/fixtures/app/views/main.html`)…

Maybe it is possible as a unit test, but with Supertest this may be avoiding the point.

Just thinking out loud here; you can probably ignore this comment 😄